test/replication/gh-5506-election-on-off.result

```diff
@@ -88,7 +88,11 @@ test_run:switch('default')
  | ---
  | - true
  | ...
-box.cfg{election_mode = 'candidate'}
+--
+--Set quorum to 1, since the replica's WAL is blocked and it can't vote for the
```

Comment:
Missing whitespace after `--` on this line.

Reply:
Thanks for the review!

I've force-pushed the following changes.

```diff

diff --git a/test/replication/election_qsync.result b/test/replication/election_qsync.result
index 3e03f43c9..ff80654ee 100644
--- a/test/replication/election_qsync.result
+++ b/test/replication/election_qsync.result
@@ -21,6 +21,9 @@ old_replication_timeout = box.cfg.replication_timeout
 old_replication = box.cfg.replication
  | ---
  | ...
+old_synchro_quorum = box.cfg.replication_synchro_quorum
+ | ---
+ | ...
 
 test_run:cmd('create server replica with rpl_master=default,\
               script="replication/replica.lua"')
@@ -176,6 +179,7 @@ box.cfg{
     replication_timeout = old_replication_timeout,                              \
     replication = old_replication,                                              \
     replication_synchro_timeout = old_replication_synchro_timeout,              \
+    replication_synchro_quorum = old_synchro_quorum,                            \
 }
  | ---
  | ...
diff --git a/test/replication/election_qsync.test.lua b/test/replication/election_qsync.test.lua
index b75e1c154..49fdb4993 100644
--- a/test/replication/election_qsync.test.lua
+++ b/test/replication/election_qsync.test.lua
@@ -6,6 +6,7 @@ old_election_timeout = box.cfg.election_timeout
 old_replication_synchro_timeout = box.cfg.replication_synchro_timeout
 old_replication_timeout = box.cfg.replication_timeout
 old_replication = box.cfg.replication
+old_synchro_quorum = box.cfg.replication_synchro_quorum
 
 test_run:cmd('create server replica with rpl_master=default,\
               script="replication/replica.lua"')
@@ -91,6 +92,7 @@ box.cfg{
     replication_timeout = old_replication_timeout,                              \
     replication = old_replication,                                              \
     replication_synchro_timeout = old_replication_synchro_timeout,              \
+    replication_synchro_quorum = old_synchro_quorum,                            \
 }
 box.ctl.demote()
 box.schema.user.revoke('guest', 'super')
diff --git a/test/replication/gh-5144-qsync-dup-confirm.result b/test/replication/gh-5144-qsync-dup-confirm.result
index 217e44412..21571fba8 100644
--- a/test/replication/gh-5144-qsync-dup-confirm.result
+++ b/test/replication/gh-5144-qsync-dup-confirm.result
@@ -8,6 +8,14 @@ engine = test_run:get_cfg('engine')
 fiber = require('fiber')
  | ---
  | ...
+
+old_synchro_quorum = box.cfg.replication_synchro_quorum
+ | ---
+ | ...
+old_synchro_timeout = box.cfg.replication_synchro_timeout
+ | ---
+ | ...
+
 --
 -- gh-5144: qsync should not write duplicate confirms for the same LSN. That
 -- could happen when first CONFIRM write is in progress, and more ACKs arrive.
@@ -157,3 +165,9 @@ box.ctl.demote()
 box.schema.user.revoke('guest', 'super')
  | ---
  | ...
+box.cfg{                                                                        \
+    replication_synchro_quorum = old_synchro_quorum,                            \
+    replication_synchro_timeout = old_synchro_timeout,                          \
+}
+ | ---
+ | ...
diff --git a/test/replication/gh-5144-qsync-dup-confirm.test.lua b/test/replication/gh-5144-qsync-dup-confirm.test.lua
index 1d6af2c62..b3e5b2ab2 100644
--- a/test/replication/gh-5144-qsync-dup-confirm.test.lua
+++ b/test/replication/gh-5144-qsync-dup-confirm.test.lua
@@ -1,6 +1,10 @@
 test_run = require('test_run').new()
 engine = test_run:get_cfg('engine')
 fiber = require('fiber')
+
+old_synchro_quorum = box.cfg.replication_synchro_quorum
+old_synchro_timeout = box.cfg.replication_synchro_timeout
+
 --
 -- gh-5144: qsync should not write duplicate confirms for the same LSN. That
 -- could happen when first CONFIRM write is in progress, and more ACKs arrive.
@@ -72,3 +76,7 @@ test_run:cmd('delete server replica2')
 
 box.ctl.demote()
 box.schema.user.revoke('guest', 'super')
+box.cfg{                                                                        \
+    replication_synchro_quorum = old_synchro_quorum,                            \
+    replication_synchro_timeout = old_synchro_timeout,                          \
+}
diff --git a/test/replication/gh-5506-election-on-off.result b/test/replication/gh-5506-election-on-off.result
index 50e580d2c..1194ba661 100644
--- a/test/replication/gh-5506-election-on-off.result
+++ b/test/replication/gh-5506-election-on-off.result
@@ -9,6 +9,9 @@ old_election_mode = box.cfg.election_mode
 old_replication_timeout = box.cfg.replication_timeout
  | ---
  | ...
+old_synchro_quorum = box.cfg.replication_synchro_quorum
+ | ---
+ | ...
 
 --
 -- gh-5506: Raft state machine crashed in case there was a WAL write in
@@ -89,7 +92,7 @@ test_run:switch('default')
  | - true
  | ...
 --
---Set quorum to 1, since the replica's WAL is blocked and it can't vote for the
+-- Set quorum to 1, since the replica's WAL is blocked and it can't vote for the
 -- master.
 --
 box.cfg{election_mode = 'candidate', replication_synchro_quorum = 1}
@@ -139,6 +142,7 @@ box.schema.user.revoke('guest', 'super')
 box.cfg{                                                                        \
     election_mode = old_election_mode,                                          \
     replication_timeout = old_replication_timeout,                              \
+    replication_synchro_quorum = old_synchro_quorum,                            \
 }
  | ---
  | ...
diff --git a/test/replication/gh-5506-election-on-off.test.lua b/test/replication/gh-5506-election-on-off.test.lua
index 38eed2da6..b936f10e5 100644
--- a/test/replication/gh-5506-election-on-off.test.lua
+++ b/test/replication/gh-5506-election-on-off.test.lua
@@ -2,6 +2,7 @@ test_run = require('test_run').new()
 
 old_election_mode = box.cfg.election_mode
 old_replication_timeout = box.cfg.replication_timeout
+old_synchro_quorum = box.cfg.replication_synchro_quorum
 
 --
 -- gh-5506: Raft state machine crashed in case there was a WAL write in
@@ -46,7 +47,7 @@ box.error.injection.set("ERRINJ_WAL_DELAY_COUNTDOWN", 0)
 
 test_run:switch('default')
 --
---Set quorum to 1, since the replica's WAL is blocked and it can't vote for the
+-- Set quorum to 1, since the replica's WAL is blocked and it can't vote for the
 -- master.
 --
 box.cfg{election_mode = 'candidate', replication_synchro_quorum = 1}
@@ -69,5 +70,6 @@ box.schema.user.revoke('guest', 'super')
 box.cfg{                                                                        \
     election_mode = old_election_mode,                                          \
     replication_timeout = old_replication_timeout,                              \
+    replication_synchro_quorum = old_synchro_quorum,                            \
 }
 box.ctl.demote()

```